source/extensions/common/wasm/wavm/wavm.cc

```diff
@@ -176,9 +178,30 @@ template <typename F> EnvoyHandlerBase* MakeEnvoyHandler(F handler) {
   return new EnvoyHandler<F>(handler);
 }
 
+struct WavmGlobalBase {
+  WAVM::Runtime::Global* global_ = nullptr;
+};
+
+template <typename T> struct WavmGlobal : Global<T>, Intrinsics::GenericGlobal<T>, WavmGlobalBase {
+  WavmGlobal(Common::Wasm::Wavm::Wavm* wavm, Intrinsics::Module& module, const std::string& name,
+             T value)
+      : Intrinsics::GenericGlobal<T>(module, name.c_str(), value), wavm_(wavm) {}
+  virtual ~WavmGlobal() {}
+
+  T get();
+  void set(const T& t);
```

Comment:
virtual void set(const T& t) override;

BTW: is multiple line comment supported? I didn't find an approach to mark multiple lines that need attention.